ts/webui/src/components/experiment/trialdetail/chart/DefaultMetricPoint.tsx

```diff
@@ -128,20 +144,36 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
     }
 
     generateBestCurveSeries(trials: Trial[]): any {
-        const { userSelectOptimizeMode } = this.state;
+        const { userSelectOptimizeMode, userSelectAccuracyNumberKey } = this.state;
         let best = trials[0];
-        const data = [[best.sequenceId, best.accuracy, best.id, best.description.parameters]];
-
+        const data = [
+            [
+                best.sequenceId,
+                best.acc === undefined ? 0 : best.acc[userSelectAccuracyNumberKey],
+                best.id,
+                best.description.parameters
+            ]
+        ];
         for (let i = 1; i < trials.length; i++) {
             const trial = trials[i];
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            const delta = trial.accuracy! - best.accuracy!;
+            const delta = trial.acc![userSelectAccuracyNumberKey] - best.acc![userSelectAccuracyNumberKey];
             const better = userSelectOptimizeMode === 'minimize' ? delta < 0 : delta > 0;
             if (better) {
-                data.push([trial.sequenceId, trial.accuracy, best.id, trial.description.parameters]);
+                data.push([
+                    trial.sequenceId,
+                    trial.acc === undefined ? 0 : trial.acc[userSelectAccuracyNumberKey],
```

Comment:
How do you make sure every trial has the key `userSelectAccuracyNumberKey`?

Reply:
if a successful trial had return the final result, i will think the final result have the right format! maybe some reason it lost the all final result and it is possible trial got the part of final result? and why cause that status?